analyzers/src/SonarAnalyzer.Common/Rules/CertificateValidationCheckBase.cs

```diff
@@ -308,7 +308,8 @@ private static ImmutableArray<TInvocationExpressionSyntax> FindInvocationList(Sy
             var ret = ImmutableArray.CreateBuilder<TInvocationExpressionSyntax>();
             foreach (var invocation in root.DescendantNodesAndSelf().OfType<TInvocationExpressionSyntax>())
             {
-                if (c.SemanticModel.GetSymbolInfo(invocation).Symbol.Equals(method))
+                if (c.SemanticModel.GetSymbolInfo(invocation).Symbol is { } symbol
```

Comment:
For performance: Let's add a syntax-based name check first.

(I'm not reviewing, I was just curious about the NRE)